Userland/Libraries/LibC/math.cpp

```diff
@@ -90,9 +95,17 @@ static FloatType internal_to_integer(FloatType x, RoundingMode rounding_mode)
             return x;
 
         auto dead_bitcount = Extractor::mantissa_bits - unbiased_exponent;
-        auto dead_mask = (1ull << dead_bitcount) - 1;
+        // Avoid shifting by the integer type's size since that's UB.
+        auto dead_mask = dead_bitcount == sizeof(typename Extractor::ComponentType) * 8 ? ~zero : (one << dead_bitcount) - 1;
         auto dead_bits = extractor.mantissa & dead_mask;
         extractor.mantissa &= ~dead_mask;
+#if AK_HAS_FLOAT_80
```

Comment:
This breaks aarch64 and riscv64, as `AK_HAS_FLOAT_80` is not defined there.
aarch64 CI also complains about this.